src/dimensions.js

```diff
@@ -19,6 +19,11 @@ jQuery.each( { Height: "height", Width: "width" }, function( name, type ) {
 
 				if ( jQuery.isWindow( elem ) ) {
 
+					// $( window ).outerWidth/Height return w/h including scrollbars (gh-1729)
+					if ( funcName.indexOf( "outer" ) === 0 ) {
+						return elem[ "inner" + name ];
+					}
+
 					// As of 5/8/2012 this will yield incorrect results for Mobile Safari, but there
```

Comment:
Does this bug still exist in Mobile Safari? If not let's remove the comment, if so I can add a `// Supports` for it. I don't have an iPhone to check and I'm not sure the emulators are reliable enough.